docs/deprecations/migrating-from-prometheus-exec-to-prometheus.md

```diff
@@ -0,0 +1,28 @@
+# Removal of deprecated prometheusexec receiever
+## Why is this happening?
+There exist [security concerns](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6722) using the promethusexec receiver.  All prometheus specific functionality should be supported in the "normal" prometheus (scraping) receiver, along with others
+
+## If I'm using the prometheusexec receiver, what should I do?
+
+We recommend you migrate your configuration to use the [prometheus](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/prometheusreceiver) opentelemetry receiver.
+
+### Natively your metrics for scraping
+The functionality and responsibility of instantiating and exposing an endpoint to be scraped from now lies with the end user, and is not supported in any of the current prometheus receivers.  The exact method of your migration
+will depend on why you needed to use the `prometheus_exec` scraper in the first place.
+
+The easiest path would likely be to [enable federation](https://prometheus.io/docs/prometheus/latest/federation/) on your prometheus server, you can see our [prometheus-federation](./examples/prometheus-federation/README.md) example under [`examples/prometheus-federation`](./examples/prometheus-federation) in this git repository.  Ensure your endpoint is accessible from your otel collector, and feel free to ask in the [#otel-prometheus-wg](https://cloud-native.slack.com/archives/C01LSCJBXDZ), [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W), or [#prometheus](https://cloud-native.slack.com/archives/C167KFM6C) slack channels for any help.  You can also feel free to cut us an issue, or reach our to your support rep.
```

Comment:
I am not familiar with this receiver, so maybe it is obvious to a user, but this kind of confuses me. I thought exec receiver would only be used if they wanted to export 3rd party format metrics as prometheus. Could we clarify when would it be better to leverage federation as a replacement for the exec recv? How does enabling federation help a user migrating away from the exec receiver?